Package.swift

```diff
@@ -5,7 +5,7 @@ import PackageDescription
 
 // MARK: - Setup
 
-private let environment: Environment = .development
+private let environment: Environment = .production
```

Comment:
I really want to avoid a persistent git diff when working on the iOS SDK locally.
1. You mentioned that environment values are not possible, but there is conflicting evidence on the internet 😄 [swift-evolution 0273](https://github.com/apple/swift-evolution/blob/main/proposals/0273-swiftpm-conditional-target-dependencies.md) mentions that "Swift 5.3 supports platform conditionals, but not configuration conditionals". [This Swift forum post](https://forums.swift.org/t/how-to-use-a-custom-build-configuration-in-a-swift-package/36259) implies that `.debug` and `.release` configurations are available (could we use these?). [This article](https://www.oreilly.com/library/view/extending-swift-values/9781492048817/ch04.html) suggests that environment values can be used - but maybe just not in `Package.swift`. [This gist](https://gist.github.com/Sorix/21e61347f478ae2e83ef4d8a92d933af) shows a concrete example of using environment values, but I have no idea if it actually works. @Ignelios @mikasdo where is the truth? 😄
1. What happens if this is `.development` by default? Is the issue that SDK users that include it via SPM get the `.development` configuration?